README.md

```diff
@@ -1,11 +1,69 @@
 # Java Chassis [![Build Status](https://travis-ci.org/ServiceComb/java-chassis.svg?branch=master)](https://travis-ci.org/WillemJiang/java-chassis)[![Coverage Status](https://coveralls.io/repos/github/WillemJiang/java-chassis/badge.svg?branch=master)](https://coveralls.io/github/WillemJiang/java-chassis?branch=master)
 ServiceComb Java Chassis is a Software Development Kit (SDK) for rapid development of microservices in Java, providing service registration, service discovery, dynamic routing, and service management features
 
+
+## Quick Start
+
+Provider service:
+```java
+import io.servicecomb.*;
+@RpcSchema(schemaId = "helloworld")
```

Comment:
Added details describing a bit of redundancy. Our users are developers who understand their basic meaning through code. Detailed development, they refer to our specific documentation can be.  we  also has the title("Provider service") for easy understand it.